### ISSUE TITLE
如何降低训练时的data_cost

### ISSUE THREAD

**Issue by qiuming-93** [PaddlePaddle/PaddleDetection#8765]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

[12/22 16:25:45] ppdet.engine INFO: Epoch: [18] [0/4] learning_rate: 0.000713 loss: 1.525035 loss_cls: 0.844636 loss_iou: 0.118553 loss_dfl: 0.769451 loss_l1: 0.231170 eta: 0:35:24 batch_cost: 8.5667 data_cost: 7.8027 ips: 2.8015 images/s
使用PaddleDetection训练ppyoloE+，如何降低data_cost，提高训练效率

**Comment by lyuwenyu** [comment]
可以把`num_worker`改大点
